avocado/core/job.py

```diff
@@ -198,7 +198,7 @@ def __exit__(self, _exc_type, _exc_value, _traceback):
 
     def __start_job_logging(self):
         # Enable test logger
-        fmt = ('%(asctime)s %(module)-16.16s L%(lineno)-.4d %('
+        fmt = ('%(asctime)s %(name)s L%(lineno)-.4d %('
```

Comment:
I don't think `name` will make sense alongside `lineno`.  AFAICT, `name` can be something like `avocado.test` and the `lineno` will not be pointing somewhere users can relate to, right?

Reply:
Yes, agreed. I would vote to remove the line number since is not useful for users, and developers can get the line number quite easily, But we can move this commit/discussion post LTS.